input/P15tSZ.yml

```diff
@@ -0,0 +1,12 @@
+mapper_class: MapperP15tSZ
+map_name: P15tSZ
+file_map: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-ymaps_2048_R2.00.fits
+file_noise: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-stddev_2048_R2.00.fits
+file_mask: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-masks_2048_R2.01.fits
+gp_mask_mode: '0.5'
+ps_mask_mode: ['default']
+beam_info: [{'type': 'Gaussian', 'FWHM_arcmin': 10.0}]
+coords: G
+custom_auto: True
```

Comment:
Not needed. All mappers inheriting from `PlanckBase` have this hard-coded: https://github.com/xC-ell/xCell/blob/11eebb08fb6d31a97032e9610e5479745afe08fa/xcell/mappers/mapper_Planck_base.py#L28

Reply:
Done!